scss/partials/_why-partner-with-us.scss

```diff
@@ -0,0 +1,26 @@
+.why-partner-with-us {
+  @include grid();
```

Comment:
If this works fine, I don't think we need to outright change it, but given how generally simple the overall layout of this component is, I wonder if we couldn't get the same outcome from just some simple padding and margining. on the inner content wrapper.